tests/test_base_close_loop.py

```diff
@@ -0,0 +1,143 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import random
+import unittest
+
+import numpy as np
+from lsst.ts import standardscripts
+from lsst.ts.observatory.control.utils.enums import ClosedLoopMode
+from lsst.ts.standardscripts import BaseCloseLoop
```

Comment:
`BaseCloseLoop` is an abstract class, I don't think you can unit test it. You will have to test the concrete implementations.

Reply:
Cool, do I need to add two unit tests? lsstcam and comcam are basically the same, is it okay if I just have an lsstcam unit test?

Reply:
one one is fine.